aleph-client/src/pallets/staking.rs

```diff
@@ -441,12 +437,14 @@ impl<C: AsConnection + Sync> StakingRawApi for C {
     ) -> Vec<StorageKey> {
         let key_addrs = api::storage().staking().eras_stakers_root();
         let mut key = key_addrs.to_root_bytes();
-        StorageMapKey::new(era, StorageHasher::Twox64Concat).to_bytes(&mut key);
+        key.extend(subxt::ext::sp_core::twox_64(&era.encode()));
+        key.extend(&era.encode());
         accounts
             .iter()
             .map(|account| {
                 let mut key = key.clone();
-                StorageMapKey::new(account, StorageHasher::Twox64Concat).to_bytes(&mut key);
+                key.extend(subxt::ext::sp_core::twox_64(&account.encode()));
+                key.extend(&account.encode());
```

Comment:
maybe it'd be convenient to create some utility function that takes `key`, `subkey` and performs:
```rust
key.extend(subxt::ext::sp_core::twox_64(&subkey.encode()));
key.extend(&subkey.encode()));
```